breacharbiter.go

```diff
@@ -1,21 +1,35 @@
 package main
 
 import (
+	"bytes"
+	"encoding/binary"
+	"errors"
 	"fmt"
+	"io"
 	"sync"
 	"sync/atomic"
 
+	"github.com/boltdb/bolt"
 	"github.com/davecgh/go-spew/spew"
 	"github.com/lightningnetwork/lnd/chainntnfs"
 	"github.com/lightningnetwork/lnd/channeldb"
 	"github.com/lightningnetwork/lnd/htlcswitch"
 	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/roasbeef/btcd/btcec"
 	"github.com/roasbeef/btcd/chaincfg/chainhash"
 	"github.com/roasbeef/btcd/txscript"
 	"github.com/roasbeef/btcd/wire"
 	"github.com/roasbeef/btcutil"
 )
 
+// retributionBucket stores retribution state on disk between detecting a
+// contract breach, broadcasting a justice transaction that sweeps the channel,
+// and finally witnessing the justice transaction confirm on the blockchain. It
+// is critical that such state is persisted on disk, so that if our node
+// restarts at any point during the retribution procedure, we can recover and
+// continue from the persisted state.
+var retributionBucket = []byte("ret")
```

Comment:
I don't have enough knowledge of other bucket names, but we could choose something more distinctive at the very least?